microsoft/testsuites/kata_conformance/kata_conformance_tests.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json
+from pathlib import Path
+from typing import Any, Dict
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.operating_system import CBLMariner, Ubuntu
+from lisa.testsuite import TestResult
+from lisa.tools import Lscpu
+from lisa.util import LisaException, SkippedException
+from microsoft.testsuites.kata_conformance.aks_infra import AKSInfra
+from microsoft.testsuites.kata_conformance.kata_conformance_tests_tool import (
+    KataConformanceTests,
+)
+
+
+def validate_variable(variables, key, log, nullable=False):
+    log.debug(f"Checking for param : {key} in variables , nullable : {nullable}")
+    value = variables.get(key, None)
+    if (not nullable and value is None) or (
```

Comment:
In LISA, the principle is to set default values as much as possible, so the test cases is easy to run. We shouldn't ask users to provide every parameter to get test case work.

Reply:
Implemented defaults.